docs/sources/flow/concepts/clustering.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Clustering
+weight: 500
+labels:
+  stage: beta
+---
+
+# Clustering (beta)
```

Comment:
Would be nice to have a comparison of clustering vs hashmod, and when to chose what in different situations?

Reply:
For now we don't want to document anything other than clustering, since we want to only recommend clustering for scaling Flow agent installs.

Reply:
I was thinking that eventual consistency might be a problem for some customers? Although that probably shouldn't be the case, since Prometheus is designed in a way that it doesn't have to give 100% accurate data. If super accurate data is what customers want, probably Prometheus isn't the way to go. But then, clustering isn't just for Prometheus :) I'm happy if we don't mention hashmod for now, but I think eventually we might have to.